### ISSUE TITLE
RPi repository hash changed ?

### ISSUE THREAD

**Issue by snizzleorg** [ejurgensen/forked-daapd#627]
running apt update I get:
```
Get:12 http://www.gyfgafguf.dk/raspbian/forked-daapd stretch/contrib armhf Packages [1287 B]                    
Err:12 http://www.gyfgafguf.dk/raspbian/forked-daapd stretch/contrib armhf Packages
  Hash Sum mismatch
  Hashes of expected file:
   - Filesize:1287 [weak]
   - SHA256:54b5bc2ef949d9261208297067258f45d55f908b6192f091e3bf9cf7a026a87a
   - SHA1:59527e5ed226eab786c59368ccbe04503cad07a0 [weak]
   - MD5Sum:80ad8816db8aafbe3172edb43dbc995a [weak]
  Hashes of received file:
   - SHA256:0337b4c98d418d30f53c3d655ea9e847e13e319f58ec1a0e0219cf98c1550451
   - SHA1:5da4bde2e222be79eee99d65fcb63b6e849cafa3 [weak]
   - MD5Sum:e3cddb7cde7d5fb683362856530e9a04 [weak]
   - Filesize:1287 [weak]
  Last modification reported: Sun, 28 Oct 2018 18:52:52 +0000
  Release file created at: Sun, 28 Oct 2018 18:52:52 +0000
```

Is this to be expected?

**Comment by ejurgensen** [comment]
Not sure why you are seing that, I don't get that error on "apt update". I also checked the sha256 of http://www.gyfgafguf.dk/raspbian/forked-daapd/dists/stretch/contrib/binary-armhf/Packages.gz, and it is indeed 54b5bc2ef949d9261208297067258f45d55f908b6192f091e3bf9cf7a026a87a, like apt expects.

I haven't made any changes to the repo recently. I also inspected it for anything suspicious, but don't see anything.
